src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,33 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. Fields `operatingWeight` and `bladeWidth` should have private access modifiers to prevent direct access from outside the class.